src/dom-to-image-more.js

```diff
@@ -97,8 +97,8 @@
         let restorations = [];
         return Promise.resolve(node)
             .then(ensureElement)
-            .then(function (clonee) {
-                return cloneNode(clonee, options, null, ownerWindow);
+            .then(function (clone) {
```

Comment:
This is NOT a typo, the intent here is to ensure that the `clone` in scope isn't hidden by this name.

Please revert these two lines